src/dashboard/src/components/administration/management/commands/set_version.py

```diff
@@ -0,0 +1,11 @@
+from django.core.management.base import BaseCommand
+
+from installer.steps import set_agent_code
+
+
+class Command(BaseCommand):
```

Comment:
What's the difference between `set_version.py` and `set_agent_code.py`?

Reply:
Argh, `set_version.py` was the old name for that command, I'm not sure how this made it in there. I'll remove...